cost/google/unattached_volumes/google_delete_unattached_volumes.pt

```diff
@@ -6,7 +6,7 @@ category "Cost"
 severity "low"
 default_frequency "daily"
 info(
-  version: "2.9",
+  version: "2.10",
```

Comment:
Should we consider scraping recommendations from this "Google Unused Volumes" policy?

Reply:
Not scraping from this policy until it can provide accurate cost data.